samples/nrf9160/serial_lte_modem/src/slm_at_icmp.c

```diff
@@ -345,6 +346,12 @@ static int ping_test_handler(const char *url, int length, int waittime,
 		return -1;
 	}
 
+	if (length > ICMP_MAX_LEN) {
```

Comment:
It would make sense to verify input parameters at the beginning of the function. In that case, you would not need to release the DNS results (and would avoid unnecessary DNS queries).

Also, I wonder if we should use logger for the output. Would it make sense to use `m_callback` instead to print the error? All in all it's a command result, and in case logs are disabled, there would be no feedback at all.

Reply:
I had a quick flash of this idea but was lazy to move the check forward :) I'll do it.
Furthermore, we have may Error that could be feedback with AT command response, just the AT command manual would increase pages by pages, too. :D